src/NHibernate/Transaction/AdoNetWithDistributedTransactionFactory.cs

```diff
@@ -1,3 +1,4 @@
+#if !NETSTANDARD2_0
```

Comment:
Please mind that `TransactionScope` is not only about distributed transaction. I believe its most frequent usage is not distributed at all. Its main purpose is to be a transaction handling pattern. It additionally supports distributed transaction with full framework, but as far as I know, not within the standard for now. Distributed transaction are dependent on MS DTC, which is an old Windows only technology. Your test with the durable ressource showcases this, since enlisting a 2PC only durable ressource always enforces promotion to distributed. Only SinglePhasePromotable durable ressource can be used without enforcing promotion to distributed, provided their are not enlisted along with another durable ressource. Database connection are normally enlisting as durable ressource. Some data provider (like Npgsql) do enlist them as volatile, but that is not a normal case. Maybe some other data provider enlist them as 2PC only durable ressource, and then those ones will not be usable with .Net Standard.

Tests about distributed transaction (those ones forcing escalation to distributed) should be disabled if not running under the full framework. (In #627, I have made a bunch of renaming for ceasing to imply distributed where nothing was distributed: this should help having a clearer view.)

But we should have all the other `TransactionScope` use cases working.